src/bit-components.js

```diff
@@ -319,3 +319,7 @@ export const LinearScale = defineComponent({
   targetY: Types.f32,
   targetZ: Types.f32
 });
+export const Quack = defineComponent({
+  quackPercentage: Types.f32,
+  specialQuackPercentage: Types.f32
+});
```

Comment:
Unnecessary to expose these as properties when I don't think we will be making entities with quack components that have different randomization ratios.

Reply:
Yes, I don't think we need this or the inflator. This is mostly a testing component that we don't really customize so we can get rid of the properties and the inflator and hardcode the defaults in the system I think.